.github/workflows/Whitespace.yml

```diff
@@ -0,0 +1,23 @@
+name: Whitespace
+
+permissions: {}
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  typos-check:
+    name: Check whitespace
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    steps:
+      - name: Checkout the JuliaLang/julia repository
+        uses: actions/checkout@v4
```

Comment:
Pin to the commit hash (instead of the tag)?

Reply:
No workflow in this repo pins to a commit hash as far as I can tell. Also, pinning is annoying without dependabot, which isn't installed here (any specific reason why not?). I'm not against doing it, but this seems a bit inconsistent with the rest.

Reply:
We indeed should to set up Dependabot here.

https://github.com/JuliaLang/julia/pull/55283 now does the following:
1. Sets up Dependabot.
2. Pins all GitHub Actions to full-length commit hashes.